data-prepper-core/src/test/java/org/opensearch/dataprepper/pipeline/PipelineConnectorTest.java

```diff
@@ -40,11 +40,14 @@
 import java.util.Date;
 import java.util.LinkedList;
 import java.util.concurrent.TimeoutException;
+
 import com.google.common.collect.ImmutableMap;
 
 
 import static org.junit.Assert.assertTrue;
+
 import org.junit.jupiter.api.Assertions;
+
 import static org.mockito.ArgumentMatchers.any;
```

Comment:
Is it possible to group all the static and non-static imports together here?

Reply:
Probably. I didn't actually modify this file other than reformatting it to make it readable. We could optimize imports.